client/src/app/game/modals/change-hermit-modal.tsx

```diff
@@ -54,32 +54,29 @@ function ChangeHermitModal({closeModal, info}: Props) {
 	}
 
 	return (
-		<Modal title="Change active hermit" closeModal={handleNo}>
-			<div className={css.confirmModal}>
-				<div className={css.description}>
-					{lastAction && (
-						<div className={css.turnEndNotification}>
-							<span className={css.infoIcon}>!</span>
-							{/* Hermit change is the last action of your turn. */}
-							<p>Changing your hermit will end your turn!</p>
-						</div>
-					)}
-					{message}
-				</div>
-
-				<div className={css.options}>
-					{canChange ? (
-						<>
-							<Button onClick={handleNo}>Cancel</Button>
-							<Button onClick={handleYes}>Yes</Button>
-						</>
-					) : (
-						<>
-							<Button onClick={handleNo}>Ok</Button>
-						</>
-					)}
-				</div>
-			</div>
+		<Modal setOpen title="Change active hermit" onClose={handleNo}>
+			<Modal.Description>
+				{lastAction && (
+					<div className={css.turnEndNotification}>
+						<span className={css.infoIcon}>!</span>
+						{/* Hermit change is the last action of your turn. */}
```

Comment:
This comment doesnt add any useful information, meaning it will probably just become wrong or something

Reply:
Not written by me, but can remove

Reply:
Turns out it was somehow added by me, but fixed now.